src/components/Settings/TwoStepVerification.vue

```diff
@@ -1,33 +1,43 @@
 <template>
-	<kv-settings-card title="2-Step verification">
-		<template v-slot:icon>
-			<!-- TODO: THIS ICON IS A PLACEHOLDER
-					Get correct icon assest from design, or remove this KvIcon -->
-			<kv-icon
-				class="icon"
-				title="Auto-lending On"
-				name="auto-icon-on"
-			/>
-		</template>
-		<template v-slot:content>
-			<h3 class="strong">
-				Status:
-				<span :class="`mfa-${MFAStatus}`">{{ MFAStatus }}</span>
-			</h3>
-			<p>
-				Protect your Kiva account with an extra layer of security by requiring access
-				to your phone. Once configured, you'll be required to enter
-				<span class="strong">both your password and an authenication code from your mobile phone</span>
-				in order to access your account.
-			</p>
-			<kv-button
-				to="/settings/security/mfa"
-				class="smallest"
-			>
-				Manage 2-step verification
-			</kv-button>
-		</template>
-	</kv-settings-card>
+	<div class="two-step-card-wrapper">
+		<kv-loading-placeholder
+			class="two-step-card-wrapper--loading"
+			v-if="isLoading"
+		/>
+		<kv-settings-card
+			v-if="!isLoading"
+			title="2-Step verification"
+			class="two-step-card-wrapper__two-step-card"
+		>
+			<template v-slot:icon>
+				<!-- TODO: THIS ICON IS A PLACEHOLDER
+						Get correct icon assest from design, or remove this KvIcon -->
+				<kv-icon
+					class="icon"
+					title="Auto-lending On"
+					name="auto-icon-on"
+				/>
+			</template>
+			<template v-slot:content>
+				<h3 class="strong">
+					Status:
+					<span :class="`two-step-card-wrapper__two-step-card--mfa-${MFAStatus}`">{{ MFAStatus }}</span>
```

Comment:
I think it would be better to have the loading placeholder be just for this MFA status span instead of the entire settings card, that way the user won't have to wait to be able to click the "Manage 2-step verification" button.

Reply:
Moved the loading placeholder to work with on/off span instead of entire card.
<img width="825" alt="Screen Shot 2021-01-07 at 10 23 21 AM" src="https://user-images.githubusercontent.com/1521381/103930633-e8dc3800-50d3-11eb-901b-bd0f8d4ed786.png">